### PR TITLE
Fix all tests

### PR DESCRIPTION
**Pull request by FredTheDino**

### CODE REVIEW

File: tests/core/string_conversion.sy [hunk 1]
```diff
@@ -59,6 +59,8 @@ start :: fn {
     print(d)
 
     // Other
-    print(A, pop, :str)
+    print' A
+    print' pop
+    print' :str
     as_str(nil) <=> "nil"
```
Comment (FredTheDino) [inline tests/core/string_conversion.sy:65]: This makes me sad...

File: sylt-common/src/ty.rs [hunk 2]
```diff
@@ -214,6 +214,14 @@ impl Type {
                         ));
                     }
                 }
+                if a_args.len() != b_args.len() {
+                    return Err(
+                        format!(
+                            "mismatching arity, {} != {}",
```
Comment (EMattfolk) [inline sylt-common/src/ty.rs:220]:
```suggestion
                            "Mismatching arity, {} != {}",
```

File: tests/core/string_conversion.sy (hunk 1 above)
Comment (EMattfolk) [reply tests/core/string_conversion.sy:65]: Not being able to print all at the same time? We could assume that they will look a specific way. Or just do a as_str instead.

File: tests/for_each/for_in_for.sy [hunk 3]
```diff
@@ -2,7 +2,7 @@ start :: fn {
     values := [1,2,3,4]
     values -> for_each' fn a:int {
         values -> for_each' fn b:int {
-            print (a, b)
```
Comment (EMattfolk) [inline tests/for_each/for_in_for.sy:5]: 🤔

**Review by EMattfolk — APPROVED** [review]
Looks good. I think it is time to rename the folder to `typing` though.

**Review by sornas — APPROVED** [review]

File: tests/core/string_conversion.sy (hunk 1 above)
Comment (FredTheDino) [reply tests/core/string_conversion.sy:65]: This is equivalent to what was there before - these changes should be done in a different pr.

File: sylt-common/src/ty.rs (hunk 2 above)
Comment (FredTheDino) [reply sylt-common/src/ty.rs:220]: This is already uniform with other errors

**Pull request merged**